mail/dovecot/Portfile

```diff
@@ -1,7 +1,9 @@
 PortSystem          1.0
+# This port can be removed on September 8, 2020. 
+PortGroup           obsolete 1.0
 
 name                dovecot
-conflicts           dovecot2
+replaced_by         dovecot2
 epoch               20060722
 version             1.2.17
 revision            2
```

Comment:
You will need to increase the `revision` by 1 so that people who have it installed will get the replacement port.